setup.py

```diff
@@ -33,7 +33,8 @@ def read(fname):
     zip_safe=False,
     python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
     install_requires=[
-        "arrow<0.14.0", "cookiecutter>=1.4.0,<1.8.0", "pytest>=3.3.0,<6.0.0"
+        "arrow<0.14.0;python_version>'3.0' and python_version<'3.5'",
```

Comment:
Why is this constraint required? If arrow >= 0.14.0 doesn't support python < 3.5, then pip will handle that itself.

Reply:
Also I think that this will not include arrow as a requirement for Python >= 3.5 and <= 3.0

Reply:
@jamesmyatt , No pip wont handle it by itself, otherwise I would have not gone to such horrid lengths to fix this.  pip only excludes versions when they were uploaded with correct `python_requires`, and that is usually not done.
We do not need an explicit rule for arrow default version.  It is included by default by cookiecutter.
These rules are only to help pip when its defaults from cookiecutter dont work.

You can see it being installed in the CI logs.

Reply:
@jayvdb OK thanks